runtime_config_linux.go

```diff
@@ -198,6 +198,12 @@ type Network struct {
 	Priorities []InterfacePriority `json:"priorities"`
 }
 
+// IntelRdt for Linux cgroup 'intel_rdt' resource management
+type IntelRdt struct {
+	// L3 cache capacity bitmask (CBM) for container
+	L3Cbm *string `json:"l3Cbm,omitempty"`
```

Comment:
>xiaochenshen added
>@others, please tell me if you have different opinions.
>Otherwise, I will change the type of IntelRdt.L3Cbm from *string to *uint64 later.

Ping @mrunalp @vishh @hqhq @philips